README.md

```diff
@@ -17,48 +17,27 @@ For more details on schema customization and use of the schema assistant, see
 capabilities of HarbourBridge.
 
 HarbourBridge is designed to simplify Spanner evaluation and migration, and in
-particular for migrating moderate-size PostgreSQL/MySQL/DynamoDB datasets to Spanner 
+particular for migrating moderate-size PostgreSQL/MySQL/DynamoDB datasets to Spanner
 (up to about 100GB). Many features of PostgreSQL/MySQL, especially those that don't
 map directly to Spanner features, are ignored, e.g. stored functions and
 procedures, and sequences. Types such as integers, floats, char/text, bools,
 timestamps, and (some) array types, map fairly directly to Spanner, but many
 other types do not and instead are mapped to Spanner's `STRING(MAX)`.
-In the case of DynamoDB, the schema is inferred based on a certain 
+In the case of DynamoDB, the schema is inferred based on a certain
 amount of sampled data.
 
 View HarbourBridge as a way to get up and running fast, so you can focus on
 critical things like tuning performance and getting the most out of
 Spanner. Expect that you'll need to tweak and enhance what HarbourBridge
 produces.
 
-To use the tool on a PostgreSQL database called mydb, run
-
-```sh
-# By default, the driver is "pg_dump".
-pg_dump mydb | harbourbridge
-# Or,
-pg_dump mydb | harbourbridge -driver=pg_dump
-```
-
-To use the tool on a MySQL database called mydb, run
-
-```sh
-mysqldump mydb | harbourbridge -driver=mysqldump
-```
-
-To use the tool on a DynamoDB database, run
-```sh
-harbourbridge -driver=dynamodb
-```
+For some quick starter examples on how to run HarbourBridge, take a look at
+[Quickstart Guide](#quickstart-guide).
 
-HarbourBridge accepts pg_dump/mysqldump's standard plain-text format, but 
-not archive or custom formats. More details on usage can be found in 
-[Example usage](#example-usage) section.
-
-HarbourBridge automatically determines the cloud project and Spanner instance to
-use, and generates a new Spanner database name (prefixed with `{driver}_` and
-today's date). Command-line flags can be used to explicitly set the Spanner
-instance or database name. See [Options](#options).
+HarbourBridge automatically determines the cloud project to use, and generates
+a new Spanner database name (prefixed with `{source}_` and today's date).
```

Comment:
I'm not sure {source} will make too much sense for first-time readers.
Suggest just deleting "prefixed with `{source}_` and today's date)".

Reply:
Agree that it's better to just say database name since how we generate the same can change over time, and then the README would become stale. Done.